pkgs/development/r-modules/default.nix

```diff
@@ -216,508 +225,534 @@ let
   # `self` is `_self` with overridden packages;
   # packages in `_self` may depends on overridden packages.
   self = (defaultOverrides _self self) // overrides;
+
   _self = { inherit buildRPackage; } //
-          import ./bioc-packages.nix { inherit self; derive = deriveBioc; } //
+          import ./bioc-packages.nix            { inherit self; derive = deriveBioc; } //
```

Comment:
Please don't mix indentation changes into a PR that changes functionality. This makes the diffs unnecessarily long and hard to read. If you want to improve the formatting, please use a separate PR.

Reply:
that looks like a mistake.